setup.py

```diff
@@ -96,26 +20,8 @@ def run_tests(self):
 PACKAGE_DATA = {'': ['README.md', 'LICENSE.txt']}
 
 setup(
-    name=PACKAGENAME,
-    version=version.pep386,
-    author=AUTHOR,
-    author_email=AUTHOR_EMAIL,
-    description=DESCRIPTION,
-    long_description=long_description,
-    long_description_content_type=LONG_DESCRIPTION_CONTENT_TYPE,
-    url=URL,
-    classifiers=[
-        'Intended Audience :: Science/Research',
-        'License :: OSI Approved :: BSD License',
-        'Operating System :: OS Independent',
-        'Programming Language :: Python',
-        'Topic :: Scientific/Engineering :: Astronomy',
-        'Topic :: Software Development :: Libraries :: Python Modules',
-        'Development Status :: 7 - Inactive',
-    ],
-    python_requires='>=3.5',
-    install_requires=INSTALL_REQUIRES,
```

Comment:
I think this line is needed to get `numpy` installed.

Reply:
But `numpy` is _not_ required as a runtime dependency, is it?  It is certainly a build (compile) dependency, and that is reflected in it being included in pyproject.toml.  I.e. there's no direct imports of `numpy` anywhere in this code.

Reply:
Ok, my bad, there is an import of `numpy`.  Fixing.

Reply:
I cannot imagine a package called "imagestats" that does not use numpy...

Reply:
>  I.e. there's no direct imports of `numpy` anywhere in this code.
`numpy` is used in C code to accept `numpy` arrays as input.

Reply:
It's explicitly imported here too

https://github.com/spacetelescope/stsci.imagestats/blob/30e5031b0f7a9f20bffb4349a71c776a94d675e7/stsci/imagestats/__init__.py#L11

so yeah, it's now in `install_requires`.